cypress/support/cypress-support/src/FireboltInvoker.js

```diff
@@ -22,7 +22,7 @@ const logger = require('../../Logger')('FireboltInvoker.js');
 
 const WEBSOCKET = 'WebSocket';
 const DEFAULT_WS_URL_PROTOCOL = 'ws://';
-const DEFAULT_WS_PORT = 9998;
+const DEFAULT_WS_PORT = 8080;
```

Comment:
Has this been tested with Mock Firebolt?

Reply:
Yes,I've tested in MFOS by executing suite runs.